tests/integration/test_integration.py

```diff
@@ -533,31 +533,6 @@ def encode_traces(self, traces):
     )
 
 
-@skip_if_testagent
-@pytest.mark.subprocess(
-    env={
-        "_DD_TRACE_WRITER_LOG_ERROR_PAYLOADS": "true",
-        "DD_TRACE_API_VERSION": "v0.5",
-        "DD_TRACE_WRITER_INTERVAL_SECONDS": "1000",
-    }
-)
-def test_trace_with_invalid_encoding_for_v05_payload():
```

Comment:
Why delete this?

Reply:
Because it tests a feature that is being reverted